### DISCUSSION TITLE
Should the logo not be draggable in the main menu?

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#18038] [Q&A] [answered]
The question is in the title. I'm not 100% sure how nice the main menu looks with the logo being draggable (since you can see the "hidden" part of the buttons. Also, with parallax, it seems to create some motion sickness, but that's probably just me(?)

https://user-images.githubusercontent.com/43073074/166140602-4278f122-3cf3-475e-ad0d-68ba89157a01.mp4

**Comment by bdach** [comment]
I was aware of this and didn't mind. The range of movement is constrained anyway.

**Comment by peppy** [comment, marked as answer]
Don't drag it if you don't like it?

**Reply by Gabixel** [reply]
Yeah.. I was just worried if that was intentional or not. I guess I can just ignore it.

**Reply by Squiddu** [reply]
It does seem a little too aggressive to me. Same sorta applies to certain buttons in the UI, they have way too aggressive animations that don't really fit
